src/Component/Symbolizer/Field/BrightnessField/BrightnessField.tsx

```diff
@@ -35,7 +35,7 @@ import { Expression } from 'geostyler-style';
 type InputProps = NumberExpressionInputProps['inputProps'];
 
 export interface BrightnessFieldProps extends InputProps {
-  brightness?: number;
+  value?: number;
```

Comment:
I would prefer the commit message to have a complete list of Components where this breaking change applies